.github/workflows/prettier.yaml

```diff
@@ -0,0 +1,83 @@
+name: Prettier Lint
+
+on:
+    push:
+        branches:
+            - main
```

Comment:
Does this only run when we push to main? I believe we also have this test run when we push to the branch as well.

Reply:
It is the same for test and lint yaml files. Prettier Lint runs when pushed to the branch.